components/BuildPage/WhatBuildersAreSaying.jsx

```diff
@@ -0,0 +1,123 @@
+    title: (
+      <>
+        Head of Product, Brainbot <br />
+        Core Contributor to Shutter
+      </>
+    ),
+    builderIcon: 'shutter-avatar.png',
+  },
+  {
+    projectImage: 'alterscope.png',
+    projectUrl: 'https://www.alterscope.org/',
+    quote:
+      'Olas has solved a longstanding program to bring data to a database in a decentralized, verifiable, and trusted manner. Olas is such a perfect solution that we feel like Olas has been built for us at Alterscope!',
+    name: 'Marijo Radman',
+    nameUrl: 'https://x.com/Marijo_Radman',
+    title: 'CEO, Alterscope',
+    builderIcon: 'alterscope-avatar.png',
+  },
+  {
+    projectImage: 'creator-bid.png',
+    projectUrl: 'https://creator.bid/',
+    quote:
+      "We're building Creator.bid on the cutting-edge Olas stack to be one the first Olas-powered project en-route to deliver no-code ACA (AI Creator Agents) at scale. The Olas stack has proven to be a fantastic choice for us to run our agents in a decentralized fashion with off-chain consensus at scale.",
+    name: 'Philipp Kothe',
+    nameUrl: 'https://x.com/philism_',
+    title: 'CEO, Creator.bid',
+    builderIcon: 'creator-bid-avatar.png',
+  },
+  {
+    projectImage: 'alprina.png',
+    projectUrl: 'https://alprina.com/',
+    quote:
+      "At Alprina, we harness Olas's decentralized infrastructure to develop our AI stock prediction agent, benefiting from its open-source framework and multi-chain protocol for seamless integration and enhanced security.",
+    name: 'Josh Wagenbach',
+    nameUrl: 'https://x.com/joshwagenbach',
+    title: 'CEO, Alprina',
+    builderIcon: 'alprina-avatar.png',
+  },
+  {
+    projectImage: 'nevermined.png',
+    projectUrl: 'https://nevermined.io/',
+    quote:
+      'We are building on Olas to allow AI agents to get paid across the different networks that agents are deployed to. Nevermined will work to bring Olas AI agents to life through R&D and mutual integrations. The Olas stack enables all agents to be interoperable and composable with one another so that our payment mechanism can work seamlessly.',
+    name: 'Don Gossen',
+    nameUrl: 'https://x.com/dongossen',
+    title: 'CEO, Nevermined',
+    builderIcon: 'nevermined-avatar.png',
+  },
+];
+
+export const WhatBuildersAreSaying = () => (
+  <SectionWrapper
+    customClasses="lg:p-24 px-4 py-20 mt-20 border-y bg-gradient-to-t from-[#E7EAF4] to-gray-50"
+    backgroundType="NONE"
+  >
+    <h2 className={`${SUB_HEADER_CLASS} text-center mb-6 lg:mb-14`}>
+      What Builders are saying
+    </h2>
+
+    <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4 max-w-screen-xl mx-auto">
+      {quotes.map((quote, index) => (
+        <Card
+          key={index}
+          className='flex flex-col p-6 border-2 border-white rounded-2xl shadow-sm bg-white gap-4 bg-cover bg-[url("/images/card-bg.png")]'
+        >
+          <a href={quote.projectUrl} target="_blank">
```

Comment:
we should check if `projectUrl` is in quote and otherwise render just div, working fine now but it's not a good practice to render anchor tag without href, we use a semantic tag as non-semantic

can do like here, but use 'div' instead of Link
https://github.com/valory-xyz/olas-website/blob/08d612e971c072d1b2a5a4d90b0334af4e1cbb37/components/Layout/Footer.jsx#L125

Reply:
I see... Something like this?
```
const LinkTag = item.projectUrl ? 'a' : div;
        return (
          <Card key={index} className='flex flex-col p-6 border-2 border-white rounded-2xl shadow-sm bg-white gap-4 bg-cover bg-[url("/images/card-bg.png")]'>
            <LinkTag href={quote.projectUrl} target='_blank'>
              <Image src={`/images/build-page/${quote.projectImage}`} alt='Build' width={24} height={24} className='mx-auto' />
            </LinkTag>
            ```

Reply:
yeah, but `const LinkTag = quote.projectUrl ? 'a' : 'div';`

Reply:
ideally it's actually something like this:
```
const LinkTag = quote.projectUrl ? 'a' : div;
const linkProps = quote.projectUrl ? { href: quote.projectUrl, target: '_blank' } : {};

return (
          <Card key={index} className='flex flex-col p-6 border-2 border-white rounded-2xl shadow-sm bg-white gap-4 bg-cover bg-[url("/images/card-bg.png")]'>
            <LinkTag {...linkProps}>
              <Image src={`/images/build-page/${quote.projectImage}`} alt='Build' width={24} height={24} className='mx-auto' />
            </LinkTag>

```

Reply:
ohhh